apps/web/app/api/generated-api.ts

```diff
@@ -1159,6 +1171,24 @@ export interface GetUserStatisticsResponse {
       longest: number;
       activityHistory: object;
     };
+    lastLesson: {
+      /** @format uuid */
+      id: string;
+      title: string;
+      imageUrl?: string;
+      description: string;
+      itemsCount: number;
+      itemsCompletedCount?: number;
+      lessonProgress?: "completed" | "in_progress" | "not_started";
```

Comment:
It would be good to create a type for this
`type LessonProgress = "completed" | "in_progress" | "not_started";`

Reply:
As the filename indicates, this is a generated API based on the backend schema, so I can't modify the type here 😉